zh/core-utility-libraries/xml.rst

```diff
@@ -1,18 +1,160 @@
 Xml
 ###
 
+.. php:namespace:: Cake\Utility
+
 .. php:class:: Xml
 
-.. note::
-    The documentation is not currently supported in zh language for this page.
+``class Cake\Utility\Xml``
+
+Xml类充许将数组转换成 ``SimpleXMLElement`` 或者 ``DOMDocument`` 对象，并且可以转回数组。
+
+创建XML对象
+===========
+
+.. static Cake\Utility\Xml::build($input, array $options =[])
+
+你可使用Xml::build() 载入XML-ish数据。取决于你 ``$options`` 的参数，这个方法允许你返回一个 ``SimpleXMLElement (default) or DOMDocument`` 对象，在各种数据源中，你可使用 ``Xml::build()`` 去创建一个XML对象。例如，你可以通过下面的设置创建XML::
+
+    $text = '<?xml version="1.0" encoding="utf-8"?>
+    <post>
+        <id>1</id>
+        <title>Best post</title>
+        <body> ... </body>
+    </post>';
+    $xml = Xml::build($text);
+    
+你也可以通过本地文件创建Xml对象::
+
+    //本地文件
+    $xml = Xml::build('/home/awesome/unicorns.xml');
+
+你也可以通可一个数组创建Xml对象::
+
+    $data = [
+        'post' => [
+            'id' => 1,
+            'title' => 'Best post',
+            'body' => ' ... '
+        ]
+    ];
+    $xml = Xml::build($data);
+
+如果你的输入参数是无效的，Xml类会抛出一个异常::
+
+    $xmlString = 'What is XML?':
+    try {::
+        $xmlObject = Xml::build($xmlString); // Here will throw an exception
+    } catch (\Cake\Utility\Exception\XmlException $e) {
+        throw new InternalErrorException();
+    }
+
+.. 提示::
```

Comment:
These directives have to stay in english. Sphinx cannot process translated commands.